boards/arm/stm32l562e_dk/stm32l562e_dk.dts

```diff
@@ -31,8 +31,18 @@
 	cpu-power-states = <&stop0 &stop1 &stop2>;
 };
 
+&pinctrl {
+	usart1_lp_tx_pa9: usart1_lp_tx_pa9 {
+		pinmux = <STM32_PINMUX('A', 9, ANALOG)>;
+	};
+	usart1_lp_rx_pa10: usart1_lp_rx_pa10 {
+		pinmux = <STM32_PINMUX('A', 10, ANALOG)>;
+	};
+};
```

Comment:
lp variants should be provided by the pinctrl files, ref. https://github.com/zephyrproject-rtos/hal_stm32/blob/master/scripts/genpinctrl/stm32-pinctrl-config.yaml#L30

Reply:
Sure, that is my current though as well, but aim was to provide material for discussion, this PR is not meant for merge (cf PR description).